testbed/bin/testbed.ts

```diff
@@ -2,11 +2,13 @@
 import * as cdk from '@aws-cdk/core'
 import { Testbed } from '../stack'
 
-new Testbed(new cdk.App(), 'testbed', {
+new Testbed(new cdk.App(),
+    process.env.STACK_NAME, {
     env: {
         account: process.env.CDK_DEFAULT_ACCOUNT,
         region: process.env.CDK_DEFAULT_REGION
     },
+    stackName: process.env.STACK_NAME,
```

Comment:
Why is this being passed twice? Can't you just use the ID?

Reply:
Using the stack name which a user can pass as an environment variable, is there a way to get the ID for that?